### PR TITLE
CCD-1249 : Creating a new index for hearingVenue

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-1249

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V20210608__CCD-1249_add_index_hearingVenue.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX uidx_case_data_hearing_venue ON public.case_data USING btree (btrim(upper((data #>> '{hearingVenue}'::text[]))));
```
Comment (MSancaktutar) [inline src/main/resources/db/migration/V20210608__CCD-1249_add_index_hearingVenue.sql:1]: we should create this if not already existing.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
we should create this if not already existing.

File: src/main/resources/db/migration/V20210608__CCD-1249_add_index_hearingVenue.sql (hunk 1 above)
Comment (sumansreenivasan) [reply src/main/resources/db/migration/V20210608__CCD-1249_add_index_hearingVenue.sql]: Updated the PR

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Review by skandala-cgi — APPROVED** [review]

**Pull request merged**